### PR TITLE
doc: improve documentation

### PR DESCRIPTION
**Pull request by sdelamo**
Highlights:

- Explain better the `@OpenAPISecurity` `@OpenAPIManagement` and `@OpenAPIInclude`. Also, the need to include `open-api` in the classpath to use those.
- Add an example to the merging schemas example with an OAuth 2.0 integration with Amazon Cognito.
- Split views documentation in different sections `swagger-ui`, `redoc`, `rapidoc`, `rapipdf` and expand properties as table.
- Refactor endpoints sections.
- Group swagger annotation content.
- Try to explain better the different configuration options (properties and system properties) for OpenAPI processing.
- Split documentation in sections.
- Try to use more OpenAPI instead of Swagger.

Before and after: 

Screenshot 2021-02-10 at 17 53 19

### CODE REVIEW

File: .nova/Configuration.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (alvarosanchez) [inline .nova/Configuration.json:1]: Is this required? Or can be gitignored?

File: src/main/docs/guide/annotationConfiguration/openapimanagement.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+api:openapi.annotation.OpenAPIManagement[] adds management endpoints.
+
+api:openapi.annotation.OpenAPIManagement[] is mapped to: 
+
+```java
```
Comment (alvarosanchez) [inline src/main/docs/guide/annotationConfiguration/openapimanagement.adoc:5]: It'd be better to use asciidoctor format, not markdown

File: .nova/Tasks/Custom Task.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (alvarosanchez) [inline .nova/Tasks/Custom Task.json:1]: Same

File: src/main/docs/guide/annotationConfiguration/openapisecurity.adoc [hunk 4]
```diff
@@ -0,0 +1,9 @@
+api:openapi.annotation.OpenAPISecurity[] adds security endpoints.
+
+It is mapped to: 
+
+```java
```
Comment (alvarosanchez) [inline src/main/docs/guide/annotationConfiguration/openapisecurity.adoc:5]: asciidoctor?

**Review by alvarosanchez — CHANGES_REQUESTED** [review]

**Comment by alvarosanchez** [comment]
Some minor feedback, otherwise looks great!

File: .nova/Configuration.json (hunk 1 above)
Comment (sdelamo) [reply .nova/Configuration.json:1]: it should be ignored

File: src/main/docs/guide/annotationConfiguration/openapimanagement.adoc (hunk 2 above)
Comment (sdelamo) [reply src/main/docs/guide/annotationConfiguration/openapimanagement.adoc]: Changed

File: src/main/docs/guide/annotationConfiguration/openapisecurity.adoc (hunk 4 above)
Comment (sdelamo) [reply src/main/docs/guide/annotationConfiguration/openapisecurity.adoc]: changed

**Review by alvarosanchez — APPROVED** [review]

**Pull request merged**